impress-org/givewp
# Prevent php notices when edit donation form

Resolves https://github.com/impress-org/givewp/pull/5973#issuecomment-947971991

## Description


I find out that the admin can still see PHP notice about undefined settings in saved options if the Legacy form template selected. I updated the logic to prevent PHP notices.

## Testing Instructions


You should not see PHP notice:
- When creating a new donation form
- When editing donation form with Multi-step form template selected.
- When editing donation form with Legacy form template selected.

## Pre-review Checklist
